src/utils/muiMixout.js

```diff
@@ -0,0 +1,12 @@
+import {PropTypes} from 'react';
+import mixout, {remix, combine} from 'react-mixout';
+import pure from 'react-mixout-pure';
+import forwardContext from 'react-mixout-forward-context';
+
+export {mixout, remix, combine, pure, forwardContext};
+
+export const forwardTheme = forwardContext('muiTheme', {
+  validator: PropTypes.object.isRequired,
+});
+
+export const muiMixout = combine(pure, forwardTheme);
```

Comment:
1. Agree :+1: 
2. Yeah, and they can take advantage of the `remix` feature if they are simple enough :grin: